### PR TITLE
Remove Catbert Dependency

### PR DESCRIPTION
**Pull request by jSylvestre**
Added new tables
Need to update the views too when this is deployed because vUsers is used for Ceremony permissions

### CODE REVIEW

**Comment by jSylvestre** [comment]
Populate AllTermCodes:
USE [Commencement]
GO

INSERT INTO [dbo].[AllTermCodes]
           SELECT * from vTermCodes
GO

**Comment by srkirkland** [comment]
@jSylvestre looks good :ship:

**Comment by jSylvestre** [comment]
@srkirkland Ok, I think this PR is done. It is running on test.

File: Commencement.Core/Services/DbService.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        public DbConnection GetConnection(string connectionString = null)
+        {
+            //If connection string is null, use the default sql ce connection
+            if (connectionString == null)
+            {
+                connectionString = ConfigurationManager.ConnectionStrings["MainDB"].ConnectionString;
+            }
+
+            var connection = new SqlConnection(connectionString);
```
Comment (srkirkland) [inline Commencement.Core/Services/DbService.cs]: @jSylvestre We should try to use the reliable SQL connection from P&P like we do for give.  Might even want to think about it for the main nhibernate connection but that can be later.

**Comment by srkirkland** [comment]
@jSylvestre looks good, remember on the import scripts they need to be run on empty tables so either just run them the first time or do a truncate first.

**Pull request merged**